packages/models-library/README.md

```diff
@@ -0,0 +1,6 @@
+# simcore pydantic models library
+
+simcore library for models library
+
+## Installation
+
```

Comment:
Could you please provide more information about the scope of the library and what's meant to be stored here like you did in the PR?
It would be great for future reference.

Reply:
good point. done.